### ISSUE TITLE
nokogiri.bundle on OSX missing

### ISSUE THREAD

**Issue by Hinnack** [mitchellh/vagrant#5064]
I get:

```
/opt/vagrant/embedded/gems/gems/nokogiri-1.6.3.1/lib/nokogiri.rb:29:in `require': dlopen(/opt/vagrant/embedded/gems/gems/nokogiri-1.6.3.1/lib/nokogiri/nokogiri.bundle, 9): Library not loaded: @rpath/liblzma.dylib (LoadError)
  Referenced from: /opt/vagrant/embedded/gems/gems/nokogiri-1.6.3.1/lib/nokogiri/nokogiri.bundle
  Reason: image not found - /opt/vagrant/embedded/gems/gems/nokogiri-1.6.3.1/lib/nokogiri/nokogiri.bundle
```

when using:
config.vm.communicator = "winrm"

OSX 10.8.5
vagrant 1.7.1

**Comment by smccarthy** [comment]
brew install xz seems to fix the issue for me.

**Comment by mitchellh** [comment]
Great workaround. I think I know the issue in the meantime but that works for now.

**Comment by mitchellh** [comment]
I think I've gotten all these fixed for 1.7.2.
